src/player.cpp

```diff
@@ -4160,14 +4160,14 @@ void player::pause()
     search_surroundings();
 }
 
-void player::shout( std::string msg )
+void player::shout( std::string msg, const std::string &verbal_shout_id )
 {
-    int base = 10;
+    int base = 85;
```

Comment:
> What problem are you trying to solve?

You started this PR when I enumerated the problems with limiting NPC "signals" on map to line of sight. I assume it was supposed to solve those problems, which it doesn't.

While it can be a feature on its own, it does not "unblock" the LoS limitation PR.

> As they should be. Do you disagree with those priorities?

Once again: if this is supposed to be just an extra feature, it's OK. But it does not solve any real problem with NPCs.

> In your case 2, we already have a way to address it, #16430. 

It does nothing to address that issue. Nothing at all. Not the slightest bit.

Not being able to see the NPCs not despawn doesn't make them despawn.